.github/workflows/maven.yml

```diff
@@ -0,0 +1,27 @@
+# This workflow will build a Java project with Maven, and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Java CI with Maven
```

Comment:
@ankushdesai one thing I just noticed was that the per-OS yaml files build Rvm:

```
 24     - name: Build P runtime for Java
 25       run: |
 26         pushd Src/PRuntimes/RvmRuntime
 27         mvn install
 28         popd
```

Should we be instead doing the Maven builds for the Java PRT on a per-OS basis too?

Reply:
I would drop the Rvm compilation per OS basis and replace it with Java runtime.